pandas/tests/generic/test_generic.py

```diff
@@ -431,7 +431,7 @@ def test_size_compat(self):
 
     def test_split_compat(self):
         # xref GH8846
-        o = self._construct(shape=10)
+        o = self._construct(shape=10).values
```

Comment:
This defeats the purpose of the test, I think, as it was actually testing that `np.array_split` works on a DataFrame/Series. 

So the "quick fix" is rather to ignore warnings in this specific test, so it doesn't fail the test, I think. 

Ideally we would prevent that the numpy function raises a warning at all, but not sure that is easily possible.